code-style/scss/README.md

```diff
@@ -1,52 +1,257 @@
-Naming Conventions
-=====
+# SCSS
 
-We follow [SMACSS](http://smacss.com/) conventions when naming CSS classes.
-When you are naming something, **do not** name it based on its content, as this will be difficult to
-use the module elsewhere. Try to name modules based on function instead.
+**Table of Contents**
 
-**Modules** are the building blocks of a website. Multi-word module names should be *hyphen-separated*:
-```css
-.module {}
-/* or */
-.even-grid {}
+1. [Introduction](#Introduction)
+2. [Class Naming Theory](#Class-Naming-Theory)
+3. [BEM](#BEM)
+  1. [Using BEM](#Using-BEM)
+    1. [Block](#Block)
+    2. [Element](#Element)
+    3. [Modifier](#Modifier)
+  2. [BEM in Sass](#BEM-in-Sass)
+4. [ITCSS](#ITCSS)
+  1. [File Naming](#File-Naming)
+    1. [Settings](#Settings)
+    2. [Tools](#Tools)
+    3. [Generic](#Generic)
+    4. [Elements](#Elements)
+    5. [Objects](#Objects)
+    6. [Components](#Components)
+    7. [Utilities](#Utilities)
+5. [Name Spacing](#Name-Spacing)
+6. [ITCSS and BEM](#ITCSS-and-BEM)
+
+
+## Introduction
+
+Our CSS is organized using [Harry Roberts](csswizardry.com)' Inverted Triangle CSS (ITCSS) organizational approach. This method is mixed with [Block Element Modifier](http://getbem.com) (BEM) naming convention for class names throughout the stylesheet.
```

Comment:
I think we prefer the `[Link Text]` ... `[Link Text]: link` syntax for links so they are more readable.

Reply:
the last one here
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links